test/e2e/tests/revoke-permissions.spec.js

```diff
@@ -0,0 +1,46 @@
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  openDapp,
+  unlockWallet,
+  defaultGanacheOptions,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Wallet Revoke Permissions', function () {
+  it('should revoke eth_accounts permissions via test dapp', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+        await openDapp(driver);
+
+        // Get initial accounts permissions
+        await driver.clickElement('#getPermissions');
+
+        const permissionsResult = await driver.findElement(
+          '#permissionsResult',
+        );
+
+        // Eth_accounts permission
+        assert.equal(await permissionsResult.getText(), 'eth_accounts');
+
+        // Revoke eth_accounts permissions
+        await driver.clickElement('#revokeAccountsPermission');
```

Comment:
Looks like we need to bump `@metamask/test-dapp` to [v8.1.0](https://github.com/MetaMask/test-dapp/blob/main/CHANGELOG.md#810) where the button is added